### PR TITLE
OSDOCS-1585: Changing query and fixing example output

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OCPBUGS-1585

Link to docs preview:
https://55766--ocpdocs-pr.netlify.app/openshift-enterprise/latest/authentication/understanding-and-managing-pod-security-admission#security-context-constraints-psa-alert-eval_understanding-and-managing-pod-security-admission

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by gkarager** [comment]
@bergerhoffer Here is an example: 
$ zgrep -h pod-security.kubernetes.io/audit-violations must-gather.local.*/quay*/audit_logs/kube-apiserver/*log.gz | jq -r 'select((.annotations["pod-security.kubernetes.io/audit-violations"] != null) and (.objectRef.resource=="pods")) | .objectRef.namespace + " " + .objectRef.resource' | sort | uniq -c

Output: 
     1 example pods
     1 test pods

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.14` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.13. And any PR going into `main` must also target the latest version branch (enterprise-4.14).

If the update in your PR does NOT apply to version 4.14 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by bergerhoffer** [comment]
Hi @xingxingxia, can you please review?

Giriyamma had some updates she wanted me to make to this - the example output was wrong. But it also looked like she wanted me to remove the name of the pod in the output, but keep the resource. But that didn't make sense to me, since the resource would always be a pod, right?

So I adjusted the query to no longer show the resource type (since it's always a pod), but include the name. When I tested it, it seems like it does show the pod name fine.

So let me know what you think, thanks!

File: modules/security-context-constraints-psa-alert-eval.adoc [hunk 1]
```diff
@@ -26,17 +26,15 @@ $ oc adm must-gather -- /usr/bin/gather_audit_logs
...
 ----
-$ zgrep -h pod-security.kubernetes.io/audit-violations must-gather.local.<archive_id>/quay*/audit_logs/kube-apiserver/*log.gz \
-  | jq -r 'select((.annotations["pod-security.kubernetes.io/audit-violations"] != null) and (.objectRef.resource=="pods")) | .objectRef.namespace + " " + .objectRef.name + " " + .objectRef.resource' \
+$ zgrep -h pod-security.kubernetes.io/audit-violations must-gather.local.<archive_id>/<registry_id>/audit_logs/kube-apiserver/*log.gz \
+  | jq -r 'select((.annotations["pod-security.kubernetes.io/audit-violations"] != null) and (.objectRef.resource=="pods")) | .objectRef.namespace + " " + .objectRef.name' \
   | sort | uniq -c
 ----
 +
-Replace `must-gather.local.<archive_id>` with the actual directory name.
+Replace `<archive_id>` and `<registry_id>` with the actual path names.
```
Comment (xingxingxia) [inline modules/security-context-constraints-psa-alert-eval.adoc:34]: Minor nit: from my test example, `quay-io-openshift-release-dev-ocp-v4-0-art-dev-sha256-dda3...` looks like image, not registry. And precisely, it is not image id, is image digest `quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:dda3`'s reformat on several special characters.

Comment (bergerhoffer) [reply modules/security-context-constraints-psa-alert-eval.adoc]: Thanks @xingxingxia, I changed <registry_id> to <image_digest_id>

**Comment by rh-tokeefe** [comment]
LGTM

**Comment by bburt-rh** [comment]
Lgtm. Thanks @rh-tokeefe and @bergerhoffer

**Comment by bergerhoffer** [comment]
@bburt-rh @rh-tokeefe Thanks! Can one of you flip the peer review labels please?

**Pull request merged**